### PR TITLE
Improve defaults for Azure SQL resources + fix some branding

### PR DESCRIPTION
**Pull request by tas50**
```coffee
cnquery> azure.subscription.sql.servers.first
azure.subscription.sql.servers.first: azure.subscription.sqlService.server name="timtimtimtim" location="eastus" properties.version="12.0" properties.state="Ready" properties.fullyQualifiedDomainName="timtimtimtim.database.windows.net"

cnquery> azure.subscription.sql.servers.first.databases
azure.subscription.sql.servers.first.databases: [
  0: azure.subscription.sqlService.database name="master" edition="System" zoneRedundant=false status="Online" creationDate=2024-06-12 12:20:08.777 -0700 PDT maxSizeBytes=53687091200 earliestRestoreDate=null
]
```

### CODE REVIEW

File: providers/azure/resources/azure.lr [hunk 1]
```diff
@@ -1137,8 +1137,8 @@ private azure.subscription.sqlService.server.administrator @defaults("id name")
   tenantId string
 }
 
-// Azure SQL server database
-private azure.subscription.sqlService.database @defaults("id name") {
+// Azure SQL Database service database
```
Comment (misterpantz) [inline providers/azure/resources/azure.lr:1140]: server, not service?

File: providers/azure/resources/azure.lr [hunk 2]
```diff
@@ -1183,7 +1183,7 @@ private azure.subscription.sqlService.database @defaults("id name") {
   readScale string
   // SQL database sample name
   sampleName string
-  // Whether SQL server is zone redundant
+  // Whether the SQL is zone redundant
```
Comment (misterpantz) [inline providers/azure/resources/azure.lr]: really, just the SQL?

File: providers/azure/resources/azure.lr [hunk 3]
```diff
@@ -1199,7 +1199,7 @@ private azure.subscription.sqlService.database @defaults("id name") {
   usage() []azure.subscription.sqlService.databaseusage
 }
 
-// Azure SQL database usage
+// Azure SQL Database service database usage
```
Comment (misterpantz) [inline providers/azure/resources/azure.lr:1202]: Okay checking again: service, not server, right?

File: providers/azure/resources/azure.lr (hunk 1 above)
Comment (tas50) [reply providers/azure/resources/azure.lr:1140]: The full name is Azure SQL Database service so using the full name allows us not to have Database database. The naming of this product is problematic.

File: providers/azure/resources/azure.lr (hunk 2 above)
Comment (misterpantz) [reply providers/azure/resources/azure.lr]: This sounds like the query language is zone redundant.

Comment (tas50) [reply providers/azure/resources/azure.lr]: maybe we could make this database instead

Comment (tas50) [inline providers/azure/resources/azure.lr]:
```suggestion
  // Whether the database is zone redundant
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**